### PR TITLE
Source from aws sdk ruby as submodule

### PR DESCRIPTION
**Pull request by dupuchba**

### CODE REVIEW

**Comment by dupuchba** [comment]
Note that some specs are failing : 

- Encountered 4 errors while generating, failed for APIs: mq, connect, medialive, mediaconvert

**Comment by viesti** [comment]
`com.amazonaws/aws-java-sdk-models` dependency needs to be removed from `project.clj`

**Comment by viesti** [comment]
Christophe initially had a version that grouped multiple versions of an api into a map from which the latest version can be chosen (I think there was reliance on the iteration order returned by `file-seq`, but my memory might fail me). Since the ruby sdk has multiple versions of api specs, we need to take this into account.

**Comment by dupuchba** [comment]
It should take into account last version but I don't know why docs are now gone on the latest version

File: src/portkey/aws/kinesis_video_archived_media/_2017-09-30.clj [hunk 1]
```diff
@@ -0,0 +1,71 @@
+(ns portkey.aws.kinesis-video-archived-media.-2017-09-30 (:require [portkey.aws]))
+
+(def endpoints 'nil)
```
Comment (dupuchba) [inline src/portkey/aws/kinesis_video_archived_media/_2017-09-30.clj]: no endpoints but specs are generated...

**Comment by dupuchba** [comment]
updated paritions.json

**Pull request merged**